src/medications/ViewMedication.tsx

```diff
@@ -226,42 +224,27 @@ const ViewMedication = () => {
         <div className="border-bottom" />
         <Row>
           <Column>
-            <SelectWithLabelFormGroup
-              name="status"
-              label={t('medications.medication.status')}
-              isRequired
+            <Label title="status" text={t('medications.medication.status')} isRequired />
+            <Select
+              id="status"
               options={statusOptionsEdit}
-              defaultSelected={statusOptionsEdit.filter(
-                ({ value }) => value === medicationToView.status,
-              )}
               onChange={(values) => onFieldChange && onFieldChange('status', values[0])}
-              isEditable={isEditable}
             />
           </Column>
           <Column>
-            <SelectWithLabelFormGroup
-              name="intent"
-              label={t('medications.medication.intent')}
-              isRequired
+            <Label title="intent" text={t('medications.medication.intent')} isRequired />
+            <Select
+              id="intent"
               options={intentOptions}
-              defaultSelected={intentOptions.filter(
-                ({ value }) => value === medicationToView.intent,
-              )}
               onChange={(values) => onFieldChange && onFieldChange('intent', values[0])}
-              isEditable={isEditable}
             />
           </Column>
           <Column>
-            <SelectWithLabelFormGroup
-              name="priority"
-              label={t('medications.medication.priority')}
-              isRequired
+            <Label title="priority" text={t('medications.medication.priority')} isRequired />
+            <Select
+              id="priority"
               options={priorityOptions}
-              defaultSelected={priorityOptions.filter(
-                ({ value }) => value === medicationToView.priority,
-              )}
               onChange={(values) => onFieldChange && onFieldChange('priority', values[0])}
-              isEditable={isEditable}
```

Comment:
It seems like the isEditable field is lost here. Should we pass !isEditable as a value to the prop disabled?